src/quo2/components/avatars/group_avatar.cljs

```diff
@@ -4,25 +4,29 @@
             [react-native.core :as rn]))
 
 (def sizes
-  {:icon {:small 12
-          :medium 16
-          :large 20}
-   :container {:small 20
+  {:icon      {:small  12
+               :medium 16
+               :large  20}
+   :container {:small  20
                :medium 32
-               :large 48}})
+               :large  48}})
 
+;; TODO: this implementation does not support group display picture (can only display default group icon).
 (defn group-avatar [_]
-  (fn [{:keys [color size override-theme]}]
-    (let [theme          (or override-theme (if (colors/dark?) :dark :light))
-          container-size (get-in sizes [:container size])
-          icon-size      (get-in sizes [:icon size])]
+  (fn [{:keys [color size]}]
+    (let [container-size (get-in sizes [:container size])
+          icon-size      (get-in sizes [:icon size])
+          ;; theme          (or override-theme (if (colors/dark?) :dark :light))
```

Comment:
And @OmarBasem, do you want to keep these comments for reference in the future? Just from glancing at the code it looks like dead code. What do you think of at least writing a `NOTE: ...` comment to explain why they're there?

Reply:
This code should be working, but something is missing in the method I think, so I commented it out till it gets fixed. (opened an issue for it as well)

Reply:
Cool, so if you opened an issue we could add the issue link here in a NOTE, or TODO comment, so it doesn't look like dead code other people can delete by mistake.